### PR TITLE
Whats new for August

### PR DESCRIPTION
**Pull request by BillWagner**
Create the what's new page for August.

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and opening for a fresh build

File: docs/whats-new/dotnet-2020-08.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
```
Comment (Youssef1313) [inline docs/whats-new/dotnet-2020-08.md:1]: I believe you started to remove the "dotnet" prefix from file names.

File: docs/whats-new/dotnet-2020-08.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+- [Compiler Error CS8411](/dotnet/csharp/language-reference/compiler-messages/cs8411) - Document errors 8400, 8401, 8403, 8410, and 8411
+- [Compiler warning (level 5) CS8892](/dotnet/csharp/language-reference/compiler-messages/cs8892) - Document CS8892 warning
+- [Major language areas](/dotnet/csharp/tour-of-csharp/features) - Restructure and modernize tour of C#
+- [Program building blocks](/dotnet/csharp/tour-of-csharp/program-building-blocks) - Restructure and modernize tour of C#
+- [Types and members](/dotnet/csharp/tour-of-csharp/types) - Restructure and modernize tour of C#
+
+### Updated articles
+
+- [Compiler Warning (level 1) CS4014](/dotnet/csharp/language-reference/compiler-messages/cs4014) - Update async concept articles in the C# programming guide
+- [Sorry, we don't have specifics on this C# error](/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error) - F1 keywords
```
Comment (Youssef1313) [inline docs/whats-new/dotnet-2020-08.md:57]: This shouldn't probably be listed.

File: docs/whats-new/2020-08.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (49)
+- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
+- [JohyPark](https://github.com/JohyPark) - Joohyung Park (6)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (6)
+- [sughosneo](https://github.com/sughosneo) - Sumit Ghosh (6)
```
Comment (mairaw) [inline docs/whats-new/2020-08.md:105]: new vendor on the team

File: docs/whats-new/2020-08.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+- [bradriske](https://github.com/bradriske) (2)
+- [Mrlinkwii](https://github.com/Mrlinkwii) (2)
+- [AbregoDev](https://github.com/AbregoDev) - Daniel Moreno (1)
+- [AChehre](https://github.com/AChehre) - Ahmad Chehreghani (1)
+- [akashmanoj213](https://github.com/akashmanoj213) - Akash Manoj (1)
+- [anadem](https://github.com/anadem) - Alan Sinclair (1)
+- [angularsen](https://github.com/angularsen) - Andreas Gullberg Larsen (1)
+- [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (1)
```
Comment (mairaw) [inline docs/whats-new/2020-08.md:113]: this in theory is probably an employee behind a bot?

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: docs/whats-new/2020-08.md (hunk 4 above)
Comment (BillWagner) [reply docs/whats-new/2020-08.md]: thanks. I have made a code change to fix this, but we haven't deployed it yet.

**Pull request merged**